### ISSUE TITLE
Procedural entity handing functions need to be updated [part 2]

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#859]
...follow-up of #848:

> entity_uri and entity_label need to be updated, perhaps others too.

~~1) These functions are no longer deprecated, comments in the docblock need to be removed~~
2) Remove $entity_type, the first parameter, and create change records for these changes
3) Check the core codebase and make sure no calls to these functions pass in $entity_type

Point ! above done in parent issue in Backdrop 1.x. This one is to take care of points 2 and 3.

**Comment by serundeputy** [comment]
@klonos did you mean points 2 and 3?

**Comment by klonos** [comment]
Yep, fixed. Thanx @serundeputy Sleep deprivation kicking in. :sleepy:
